bootstrap/run_app_action.go

```diff
@@ -103,14 +103,20 @@ func (a *RunAppAction) Run(s *State) error {
 			count = 1
 		}
 		hosts := s.ShuffledHosts()
-		if a.ExpandedFormation.Release.Processes[typ].Omni {
+		proc := a.ExpandedFormation.Release.Processes[typ]
+
+		// Special case the scheduler, we only start one scheduler and allow
+		// it to start the rest of the jobs to avoid spawning too many jobs.
+		if proc.Service == "controller-scheduler" {
+			count = 1
```

Comment:
Possibly.. The way to to do it would probably be to not have the release be omni in the run-app section but have it merged in at deploy-app.